ui/src/components/HostList.scss

```
@@ -0,0 +1,33 @@
+@use "../styles/var/colors" as *;
+@use "../styles/var/fonts" as *;
+
+$font-mono: ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.13.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)